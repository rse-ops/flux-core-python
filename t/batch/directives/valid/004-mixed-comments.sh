#!/bin/sh
# Set flux directives
#FLUX: -N1
# Set job name:
#FLUX: --job-name=test
hostname; date
