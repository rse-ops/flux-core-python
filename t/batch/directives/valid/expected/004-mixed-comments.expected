SETARGS(['-N1'])
SETARGS(['--job-name=test'])
